### ISSUE TITLE
Found clangd files error

### ISSUE THREAD

**Issue by inhaledesign** [boocs/unreal-clangd#3]
When I run the ' `create unreal clangd project` command, I get the following error:

`Found clangd files while overwrite setting is: strict`

Presumably this is because I'm adding this to a project I've been working on for a while. I'm unsure how to proceed safely, tho.

**Comment by boocs** [comment]
https://github.com/boocs/unreal-clangd/tree/v2#reinstall-without-overwrite

Is this your own previous clangd project? Or are you trying to reinstall a project made by this extension?

The extension relies on settings that are created with the extension made clangd config file. If this is your personal clangd project then you'll need to backup your settings and run the create project command with the overwrite setting set to 'full'.

**Comment by inhaledesign** [comment]
Thanks for the response. It is a pre existing clangd project that I've never used with this extension.

When I go into the settings for this extension, there are none listed.

**Comment by boocs** [comment]
Make sure there is nothing in the setting's Search box. I thought you were correct and that there was some bug in the new vscode but I had something in the setting's Search box. Also just a note: Unreal 5.3 isn't supported yet. It actually works on Windows, and I should just release it on Windows, but I'm having a hard time with the Ubuntu version.

![image](https://github.com/boocs/unreal-clangd/assets/62588629/791e11d3-eb9d-45ed-bdcf-64cd055abca6)

![image](https://github.com/boocs/unreal-clangd/assets/62588629/f3874c34-7b70-40d1-a9b8-94930f9766e0)

**Comment by inhaledesign** [comment]
Yeah, I've made sure the setting search is clear and Unreal Clangd doesn't even show in the list of extensions.

![Screenshot from 2023-09-09 12-28-49](https://github.com/boocs/unreal-clangd/assets/6456065/27b94757-c357-4e13-b88a-26f2b322cc9b)

Bummer about not working with 5.3 on Ubuntu, but it doesn't surprise me. Unreal on Linux is an exercise in beating one's head against the wall.

**Comment by boocs** [comment]
Weird. Are you using an old version of VSCode? When creating the extension you can set the minimum VSCode version so maybe that's it.

I'm racking my brain trying to think of other things though!

**Comment by inhaledesign** [comment]
Don't think so, I just ran an update. I'm out and about right now so
unfortunately I can't check the version number.

**Comment by boocs** [comment]
Also check the extensions tab. If the clangd extension is disabled then this extension will also be disabled.

**Comment by inhaledesign** [comment]
Had to restart my computer but the settings are showing now.

**Comment by boocs** [comment]
Just for info, it was my fault for the Ubuntu errors but it did lead me to totally fix Ubuntu without any weird hack preinclude.
